TMPro/TextMeshPro.cs

```diff
@@ -3276,6 +3276,7 @@ private bool ValidateHtmlTag(int[] chars, int startIndex, out int endIndex)
 					}
 					m_htmlColor = HexCharsToColor(m_htmlTag, num);
 					return true;
+				case 6691:
```

Comment:
What's up with this?

Reply:
Oh yeah that's part of TMP's tag parser, it apparently decides what to do by hashing the tags and then just comparing known hash values.  6691 is the hash of `line-height` (and 7840 is the hash of `/line-height`) which is natively supported by the parsers of Watanagashi+ but not by Onikakushi's.  The other option is finding inputs that match the original hash, but then they'd have to be different for Watanagashi+ and also the only ones I could find looked like random garbage, so I though it would make more sense to just add support for using the tag `line-height`